test/deb/lima.yaml

```diff
@@ -0,0 +1,69 @@
+# This template requires Lima v0.7.0 or later.
+images:
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+memory: "6144MiB"
+cpus: 4
+
+mounts:
+- location: "/tmp/lima"
+  writable: true
+containerd:
+  system: false
+  user: false
+provision:
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    test -e /etc/kubernetes/admin.conf && exit 0
+
+      # Package configs
+      . /tmp/lima/env ||:
+      KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.28}
```

Comment:
We need to change dependencies.yaml for this: https://github.com/cri-o/packaging/blob/c9e58da106e99dce6f3ec9835b4ae2a422137d78/dependencies.yaml#L3